packages/fx-core/src/component/driver/tools/installDriver.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { StepDriver } from "../interface/stepDriver";
+import { Service } from "typedi";
+import { DriverContext } from "../interface/commonArgs";
+import { FxError, Result } from "@microsoft/teamsfx-api";
+import { wrapRun } from "../../utils/common";
+import { hooks } from "@feathersjs/hooks/lib";
+import { addStartAndEndTelemetry } from "../middleware/addStartAndEndTelemetry";
+import { TelemetryConstant } from "../../constant/commonConstant";
+import { InstallToolArgs } from "./interfaces/InstallToolArgs";
+import { LocalCertificateManager } from "../../../common/local/localCertificateManager";
+import { InvalidParameterUserError } from "./error/invalidParameterUserError";
+
+const ACTION_NAME = "tools/install";
+const outputName = Object.freeze({
+  SSL_CRT_FILE: "SSL_CRT_FILE",
+  SSL_KEY_FILE: "SSL_KEY_FILE",
+});
+const helpLink = "https://aka.ms/teamsfx-actions/tools/install";
+
+@Service(ACTION_NAME)
+export class ToolsInstallDriver implements StepDriver {
+  @hooks([addStartAndEndTelemetry(ACTION_NAME, TelemetryConstant.PROVISION_COMPONENT_NAME)])
```

Comment:
why provision component?

Reply:
Sorry, moved it to deploy life cycle.

Reply:
Just curious - it seems some drivers use actionName as component, while others use provision/deploy... Any rule?

Reply:
> Just curious - it seems some drivers use actionName as component, while others use provision/deploy... Any rule?

As @blackchoey discussed offline, there will be another story to unify the driver telemetry. Before telemetry format is decided, we can use `ACTION_NAME` now.